frontend/tests/playwright/product/add_to_cart.spec.ts

```diff
@@ -90,9 +90,7 @@ test.describe.serial('product page add to cart', () => {
       await cartDrawer.assertCartTotalQuantity(0);
       await cartDrawer.assertItemIsNotPresent(sku);
       await page.getByTestId('back-to-shopping').click();
-      await expect(
-         page.getByTestId('cart-drawer-item-count')
-      ).not.toBeVisible();
+      expect(await cartDrawer.getDrawerStatus()).toBe('closed');
```

Comment:
What do you think about `await cartDrawer.assertDrawerClosed()`?

Reply:
Should we make a generic assertion with the status we expect to be a parameter? 

Basically, if we add `assertDrawerClosed` should we also add `assertDrawerOpen` or should we make a generic one like `assertDrawerStatus('closed')`?

Reply:
> if we add assertDrawerClosed should we also add assertDrawerOpen 

Sorry, I wasn't clear. Yes.

Reply:
Addressed in [df98238](https://github.com/iFixit/react-commerce/pull/1268/commits/df982384abf7f15cc3213d911219aa48f1a9bb06)